src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -112,6 +139,24 @@ protected override void OnPersist(ApplicationEngine engine)
             var account = Contract.CreateSignatureRedeemScript(pubkey).ToScriptHash();
             GAS.Mint(engine, account, gasPerBlock * CommitteeRewardRatio / 100);
 
+            // Record the cumulative reward of the voters of each committee
+
+            var voterRewardPerCommittee = gasPerBlock * VoterRewardRatio / 100 / (ProtocolSettings.Default.CommitteeMembersCount + ProtocolSettings.Default.ValidatorsCount);
+            var committeeVotes = GetCommitteeVotes(engine.Snapshot).ToArray();
+            for (var i = 0; i < committeeVotes.Length; i++)
+            {
+                if (committeeVotes[i].Votes > 0)
+                {
+                    BigInteger voterSumRewardPerNEO = (i < ProtocolSettings.Default.ValidatorsCount ? 2 : 1) * voterRewardPerCommittee * 10000L / committeeVotes[i].Votes; // Zoom in 10000 times, and the final calculation should be divided 10000L
```

Comment:
when votes is more than 1000,0000 and there is only 1 person votes, the result is incorrect. the data need to be more accurate.